unearthed-arcana/2022/20220308.xml

```diff
@@ -0,0 +1,967 @@
+			<h4>Kender Traits</h4>
+			<p></p>
+			<p class="indent"><strong><em>Creature Type.</em></strong> You are a Humanoid.</p>
+			<p class="indent"><strong><em>Size.</em></strong> You are Small.</p>
+			<p class="indent"><strong><em>Speed.</em></strong> Your walking speed is 30 feet.</p>
+			<p class="indent"><strong><em>Brave.</em></strong> You have advantage on saving throws you make to avoid or end the frightened condition on yourself.</p>
+			<p class="indent"><strong><em>Kender Ace.</em></strong> Starting at 3rd level, you possess a magical ability to pull an item out of a bag or another container; as a bonus action, you can reach into a container you’re carrying and roll on the Kender Aces table to determine what item you pull out. The object glimmers softly and disappears after 1 hour. You can use this bonus action a number of times equal to your proficiency bonus, and you regain all expended uses when you finish a long rest.</p>
+			<p class="indent"><strong><em>Taunt.</em></strong> You have a supernatural ability to home in on a creature’s emotional raw nerves and craft a taunt that flusters that creature. As a bonus action, you can unleash a barrage of insults at a creature within 60 feet of you that can hear and understand you. The target must succeed on a Wisdom saving throw (DC equal to 8 + your proficiency bonus + your Charisma modifier), or it has disadvantage on attack rolls until the start of your next turn.</p>
+			<p class="indent">You can use this bonus action a number of times equal to your proficiency bonus, and you regain all expended uses when you finish a long rest.</p>
+			<h5>Kender Aces</h5>
+			<table>
+				<thead>
+					<tr><td>d6</td><td>Item</td></tr>
+				</thead>
+				<tr><td>1</td><td>5d6 gold pieces</td></tr>
+				<tr><td>2</td><td>1 simple weapon of your choice that has the light property</td></tr>
+				<tr><td>3</td><td>1 item of your choice from the Adventuring Gear table in the Player’s Handbook. The item must cost no more than 1 gp and weigh no more than 1 lb.</td></tr>
+				<tr><td>4</td><td>1 random item from the Trinkets table in the Player’s Handbook.</td></tr>
+				<tr><td>5</td><td>Your choice of a crowbar or a grappling hook</td></tr>
+				<tr><td>6</td><td>1 item of your choice from the Tools table in the Player’s Handbook. The item must cost no more than 10 gp.</td></tr>
+			</table>
+		</description>
+		<sheet display="false" />
+		<rules>
+			<select type="Ability Score Improvement" name="Ability Score Improvement Option (Kender)" supports="Custom Ability Score Improvement Option" />
+			<grant type="Size" id="ID_SIZE_SMALL" />
+			<stat name="innate speed" value="30" bonus="base" />
+
+			<select type="Ability Score Improvement" name="Ability Score Improvement Option (Kender)" supports="Custom Ability Score Improvement Option" />
```

Comment:
There are two selects for Ability Score Improvement, I don't think we need the second one